hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerScanHelper.java

```diff
@@ -107,6 +109,18 @@ public void handleUnhealthyScanResult(long containerID, ScanResult result) throw
     }
   }
 
+  public void triggerVolumeScan(ContainerData containerData) {
+    HddsVolume volume = containerData.getVolume();
+    if (volume != null && !volume.isFailed()) {
+      log.info("Triggering a volume scan for volume [{}] as unhealthy container [{}] was on it.",
+          volume.getStorageDir().getPath(), containerData.getContainerID());
```

Comment:
nit. `HddsVolume` already defines a `toString` with this behavior.
```suggestion
          volume, containerData.getContainerID());
```